### ISSUE TITLE
nvda does not read the deleted email address in the email app 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12226]
### Steps to reproduce:
open a new email in the windows 10 email app
in the field
for
enter an email address
now delete this email with backspace. 
### Actual behavior:
the nvda read right to left 
### Expected behavior:
nvda should read the deleted email address 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-22126,ff79f518

#### Windows version:
10 19043.899
#### Name and version of other software in use when reproducing the issue:
HxOutlook.exe, microsoft.windowscommunicationsapps 16005.13426.20688.0

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no
log
[email2.txt](https://github.com/nvaccess/nvda/files/6207130/email2.txt)

**Comment by lukaszgo1** [comment]
@fernando-jose-silva Would you be able to try this with 2020.4? It'll help to asses if this is a regression or not.

**Comment by fernando-jose-silva** [comment]
I tested with 2020.4 and the problem is present, it didn’t seem to be regression.
Thanks.

**Comment by Adriani90** [comment]
The Windows native email app is discontinued and will reach end of life by end of December this year. aFter that suers will not be able to send or receive email messages in this app. Closing as abandoned.
